README.md

```diff
@@ -81,58 +81,68 @@ These final equations are the ones used in the function **compute_bz**, which ca
 
 ## Installation 
 
-First, clone the repository
+- Clone the repository
 
 ```
 git clone https://github.com/shimming-toolbox/susceptibility-to-fieldmap-fft.git
+cd susceptibility-to-fieldmap-fft
 ```
 
-Navigate to the project directory
+- Create a virtual environnement 
 
 ```
-cd susceptibility-to-fieldmap-fft
+conda create --name <name of your environement> python=3.9
+conda activate <name of your environment>
+```
+
+- Go to the develop branch (temporary)
+
+```
+git checkout develop
 ```
 
-Install the requirements
+- Install the package
 
 ```
-pip install -r requirements.txt
+pip install .
 ```
 
+You will need to ```conda activate <name of your environment>``` each time you want to use the package.
+
 ## Usage
-To execute the scripts, you need to naviagte into the fft_simulation folder.
+Once the package is installed, the commands can be run directly from the terminal. Here is the description of the two commands available.
 
-### Analytical cases
+### analytical_cases
 
-The _analytical_case.py_ script allows for comparaison between simulated and analytical results for a spherical and cylindrical phantom. 
+The _analytical_cases_ command allows for comparaison between simulated and analytical results for a spherical and cylindrical phantom. 
 
-**Arguments** 
-- -t, type : 'spherical' or 'cylindrical'
+**Options** 
+- -t, geometry type : 'spherical' or 'cylindrical'
 - -b, buffer (optional, default=2): Buffer value for zero-padding around the phantom
 
 **Return** 
 Plots to visialize the results
 
 Example:
 ```
-python analytical_cases.py -t "spherical"
+analytical_cases -t "spherical"
 ```
 
 
-### fft_simulation
+### compute_fieldmap
```

Comment:
I would definitely place this one before. The logical order is to first compute the fieldmap, and _then_ run analytical_cases, no?